src/Http/Http.Results/tools/ResultsOfTGenerator/Program.cs

```diff
@@ -482,7 +482,7 @@ static void GenerateTest_ExecuteResult_ExecutesAssignedResult(StreamWriter write
     static void GenerateTest_Throws_ArgumentNullException_WhenHttpContextIsNull(StreamWriter writer, int typeArgNumber)
     {
         //[Fact]
-        //public void ResultsOfTResult1TResult2_Throws_ArgumentNullException_WhenHttpContextIsNull()
+        //public async Task ResultsOfTResult1TResult2_Throws_ArgumentNullException_WhenHttpContextIsNull()
```

Comment:
Does this sample test need the `(ChecksumResult1)null` change too?

Reply:
Not necessarily. This one is testing the exception that is thrown when no HTTP context is provided ([ref](https://github.com/dotnet/aspnetcore/blob/bc119fc6fd18a68f90a8672959280698ce46577a/src/Http/Http.Results/src/ResultsOfT.Generated.cs#L42)) which happens before we asset on the `IResult` type being null.